stories/components/ButtonLink/default.md

```diff
@@ -62,6 +62,7 @@ The following props are available to Button Link:
 - disabled
 - fullWidth
 - icon
+- trailingIcon
```

Comment:
Should we also add a code snippet, like you did for button [here](https://github.com/lifeomic/chroma-react/pull/193/files#diff-a8992073ddd243da6243f9f0c83f59e8a287d5fb092482704783cb614e01b1efR43)?